ajax/libs/vega/package.json

```diff
@@ -13,8 +13,13 @@
     {
       "basePath": "",
       "files": [
-        "vega.js",
-        "vega.min.js"
+        "vega*.+(js|js.map)"
+      ]
+    },
+    {
+      "basePath": "build",
+      "files": [
+        "vega*.+(js|js.map)"
```

Comment:
That should be fine. Unfortunately, I had troubled checking out the whole repo (even with partial checkouts) so I had to make my changes using on github online. This makes it hard for me to update this pr. Can you just push these changes directly?